src/components/InstrumentConfig.vue

```diff
@@ -115,6 +115,7 @@
             />
             <div v-for="opticalElementGroup in availableOpticalElementGroups" :key="opticalElementGroup.type">
               <custom-select
+                v-if="!simpleInterface || (simpleInterface && selectedinstrument !== '2M0-SCICAM-MUSCAT')"
```

Comment:
This might be a little clearer if phrased "show unless muscat is selected in the simple interface": `v-if="!( simpleInterface && selectedinstrument == '2M0-SCICAM-MUSCAT' )"`, but that's just a minor preference.

Reply:
Yeah I was considering doing it that way but didn't have much of a preference... will switch it